op-node/rollup/clsync/clsync.go

```diff
@@ -162,6 +164,28 @@ func (eq *CLSync) PublishAttributes(ctx context.Context, l2head eth.L2BlockRef)
 		return fmt.Errorf("error preparing payload attributes: %w", err)
 	}
 
+	// If our next L2 block timestamp is beyond the Sequencer drift threshold, then we must produce
+	// empty blocks (other than the L1 info deposit and any user deposits). We handle this by
+	// setting NoTxPool to true, which will cause the Sequencer to not include any transactions
+	// from the transaction pool.
+	attrs.NoTxPool = uint64(attrs.Timestamp) > l1Origin.Time+eq.spec.MaxSequencerDrift(l1Origin.Time)
+
+	// For the Ecotone activation block we shouldn't include any sequencer transactions.
+	if eq.cfg.IsEcotoneActivationBlock(uint64(attrs.Timestamp)) {
```

Comment:
so do we not add transactions for the activation block of every fork?

Reply:
yes it seems so. here is where this snippet comes from.
https://github.com/flashbots/optimism/blob/0d3273e267802599399ae99426a44a8dedfd8444/op-node/rollup/driver/sequencer.go#L97